xml/System.Threading.Tasks/ValueTask.xml

```diff
@@ -847,7 +847,13 @@ Two <xref:System.Threading.Tasks.ValueTask> instances are equal when they wrap t
       <Docs>
         <summary>Gets a <see cref="T:System.Threading.Tasks.ValueTask" /> that may be used at any point in the future.</summary>
         <returns>The preserved <see cref="T:System.Threading.Tasks.ValueTask" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+## Remarks
+
+Because a <xref:System.Threading.Tasks.ValueTask> can be backed by a pooled <xref:System.Threading.Tasks.Sources.IValueTaskSource>, it is not safe to await an arbitrary <xref:System.Threading.Tasks.ValueTask> multiple times. You can use the <xref:System.Threading.Tasks.ValueTask.Preserve> method to convert this <xref:System.Threading.Tasks.ValueTask> into the instance backed by a regular <xref:System.Threading.Tasks.Task> that is safe to await multiple times.
+]]></format>
```

Comment:
Can you please add a sentence about the difference to `AsTask`-method? Or why `Preserve` should / could be used over `AsTask`?
(It's implemented via AsTask.)

Reply:
Interesting point. Looks like an advantage of Preserve is only that it's cheaper when ValueTask represents a synchronously completed task. Preserve is nop in this case, while AsTask always constructs a new task. But it's only relevant for generic ValueTask, non-generic just uses singleton CompletedTask. I'll add this info to PR.